cgc/kmeans.py

```diff
@@ -162,77 +170,20 @@ def _compute_statistic_measures(self):
                 Z = self.Z[rr, cc]
 
                 idx = np.ravel_multi_index(
-                    (ir, ic),
-                    (len(row_clusters), len(col_clusters))
-                )
+                    (ir, ic), (len(row_clusters), len(col_clusters)))
 
                 self.stat_measures[idx, 0] = Z.mean()
                 self.stat_measures[idx, 1] = Z.std()
                 self.stat_measures[idx, 2] = np.percentile(Z, 5)
                 self.stat_measures[idx, 3] = np.percentile(Z, 95)
                 self.stat_measures[idx, 4] = Z.max()
-                self.stat_measures[idx, 4] = Z.min()
+                self.stat_measures[idx, 5] = Z.min()
```

Comment:
@fnattino When I was looking into the unit test, I found this minor bug. It is causing one column of the statistics to be always 0. Just fixed this.

Reply:
Uh great that you spotted it, thanks!